src/state/queries/messages/list-converations.ts

```diff
@@ -45,7 +46,12 @@ export function useUnreadMessageCount() {
       .flatMap(page => page.convos)
       .filter(convo => convo.id !== currentConvoId)
       .reduce((acc, convo) => {
-        return acc + (!convo.muted && convo.unreadCount > 0 ? 1 : 0)
+        const otherMember = convo.members.find(
+          member => member.did !== currentAccount?.did,
+        )
+        const blocked =
+          otherMember?.viewer?.blockedBy || otherMember?.viewer?.blocking
```

Comment:
Should also check for `blockedByList` here. I think that's it, but peep the types for `ViewerState` just in case